src/lib.rs

```diff
@@ -1,7 +1,9 @@
+#![allow(incomplete_features)] // Used for generic_const_exprs
```

Comment:
You don't strictly need this for the driver itself, and the examples (that is, if you use const expressions there) can be adjusted to not use it, so please remove. Reasoning: the feature is incomplete, and using incomplete features is dangerous.

Reply:
I think this was left over after using the arithmetic consts feature which I didn't end up using because of some compiler crashes :grimacing: